support/windows/Servo.wxs.mako

```diff
@@ -117,10 +117,154 @@
                     Name="avutil-55.dll"
                     Source="C:\msys64\mingw64\bin\avutil-55.dll"
                     DiskId="1"/>
-              <File Id="AVFormat"
-                    Name="avformat-57.dll"
-                    Source="C:\msys64\mingw64\bin\avformat-57.dll"
-                    DiskId="1"/>
+	      <File Id="AVFORMAT"
+		    Name="AVFORMAT-57.dll"
+		    Source="c:\msys64\mingw64\bin\AVFORMAT-57.DLL"
+		    DiskId="1"/>
+	      <File Id="AVUTIL"
+		    Name="AVUTIL-55.dll"
+		    Source="c:\msys64\mingw64\bin\AVUTIL-55.DLL"
+		    DiskId="1"/>
+	      <File Id="LIBBLURAY"
+		    Name="LIBBLURAY-1.dll"
+		    Source="c:\msys64\mingw64\bin\LIBBLURAY-1.DLL"
+		    DiskId="1"/>
+	      <File Id="LIBCELT0"
+		    Name="LIBCELT0-2.dll"
+		    Source="c:\msys64\mingw64\bin\LIBCELT0-2.DLL"
+		    DiskId="1"/>
+	      <File Id="LIBDCADEC"
+		    Name="LIBDCADEC-0.dll"
+		    Source="c:\msys64\mingw64\bin\LIBDCADEC-0.DLL"
+		    DiskId="1"/>
+	      <File Id="LIBFFI"
+		    Name="LIBFFI-6.dll"
+		    Source="c:\msys64\mingw64\bin\LIBFFI-6.DLL"
+		    DiskId="1"/>
+	      <File Id="LIBGMP"
+		    Name="LIBGMP-10.dll"
+		    Source="c:\msys64\mingw64\bin\LIBGMP-10.DLL"
+		    DiskId="1"/>
+	      <File Id="LIBGNUTLS"
```

Comment:
I'm guessing that ffmpeg has a networking stack and somehow we've linked everything?